knowledge-base/modify-html5-viewer-area-width-through-kendo-splitter.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Modify default Document Map and Parameter areas width in Html5 Report Viewer
+description: Resize the default areas sizes through the splitter
+type: how-to
+page_title: Change Side areas sizes in Html5 Viewer with the Kendo splitter
```

Comment:
Sometimes you mention "side areas" sometimes "default areas" and sometimes "document map and parameter areas". Instead of achieving better google search result rankings, I believe such inconsistencies would only be confusing for our users. The Title is the main thing here. The Page Title is shown in the browser tab and should be the same or shorter than the Title. The Description should expand on the Title meaning and should be slightly longer. But all three should carry the same meaning and use consistent technical terms which are not confusing.